### PR TITLE
rg-61: Wire up main page

### PR DESCRIPTION
**Pull request by Valentyn13**
> ### **TASK IS NOT READY YET**

I need more components and backend logic to implement full view of the main page
Frontend:

- Filter component

Backend:

- Crud resumes(to get all user's resumes)
- Recently viewed

### CODE REVIEW

**Review by apofis96 — APPROVED** [review]

**Review by nickolay-grechkin — APPROVED** [review]

**Comment by nickolay-grechkin** [comment]
Resolve conflicts

**Comment by nickolay-grechkin** [comment]
> This task is not ready now

If task is not ready it is better to create draft pr, so no one can merge it by mistake, also reviewers shouldn't spent time reviewing draft prs.

File: frontend/src/bundles/main-page/main-page.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                >
+                    <Spinner />
+                </div>
+            )}
+            <Header>
+                <NavTabs items={navbarItems} />
+                <UserProfile
+                    image={
+                        user
+                            ? user.user_profile.avatar
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/main-page/main-page.tsx:50]: Move this logic into a separate function

File: frontend/src/bundles/common/helpers/get-user-avatar.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { type UserWithProfileRelation } from 'shared/build';
+
+const getUserAvatart = (user: UserWithProfileRelation | null): string => {
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/helpers/get-user-avatar.ts:3]:
```suggestion
const getUserAvatartUrl = (user: UserWithProfileRelation | null): string => {
```

**Pull request merged**